azure/packages/azure-client/src/index.ts

```diff
@@ -11,6 +11,7 @@
 
 export { AzureAudience } from "./AzureAudience";
 export { AzureClient } from "./AzureClient";
+// @deprecated
```

Comment:
This comment shouldn't be needed. The `@deprecated` flag in the class's tsdoc comments should propagate to consumers using it.